docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -133,6 +140,21 @@ The cluster manager will auto-start and configure itself with credentials suppli
 
 The cluster manager exposes its REST API from the container so this can be used externally to add/remove Couchbase clusters. We also support running any scripts found in `/etc/healthcheck` to do it.
 
+=== Couchbase Cluster
```

Comment:
Let us pull this at root level which is at the same level as that of deployment and title it as:
== Connecting Couchbase
After you have deployed...is to connect the cluster to actuaqlly observe the cluster.
Two ways:

Verification: